pkg/ebpfsyncer/ebpfsyncer.go

```diff
@@ -87,6 +87,12 @@ func (e *ebpfSingleton) SyncInterfaceIngressRules(
 		return err
 	}
 
+	// Ensure IngNodeFwController's pinned links and our managed interfaces align
+	// TODO: refactor to not have managed interfaces names from two sources
+	for _, linkName := range e.c.GetPinnedLinkNames() {
```

Comment:
why we need this `createNewManager` will load pinned map ?

Reply:
pls make sure with any changes to this package u run this manual ut from ebpfsyncer folder
`ebpfsyncer]$ go test -exec sudo ./...`